src/components/tabs/js/tabDirective.js

```diff
@@ -21,6 +21,7 @@
  *
  * @param {string=} label Optional attribute to specify a simple string as the tab label
  * @param {boolean=} ng-disabled If present and expression evaluates to truthy, disabled tab selection.
+ * @param {string=} md-tab-class Optional attribute to specify a class name for the tab button
```

Comment:
Let's say "Optional attribute to specify a class that will be applied to the tab's button".